panoptes_aggregation/reducers/shape_metric_IoU.py

```diff
@@ -101,7 +107,12 @@ def IoU_metric(params1, params2, shape):
     if union == 0:
         # catch divide by zero (i.e. cases when neither shape has an area)
         return numpy.inf
-    return 1 - intersection / union
+
+    if 'temporal' in shape:
+        # combine the shape IoU with the time difference and normalize
+        return 0.5 * ((1 - intersection / union) + numpy.abs(params1[-1] - params2[-1]))
```

Comment:
Are the temporal values normalized in any way (e.g. using "frames apart" rather than seconds)?  This metric might be prone to different results dependent on what unit is used for t if it is not normalized.

Reply:
This is important here because the power of the IoU distance comes from it being "unitless" and as a result scale independent.

Reply:
I want to make a note here that the temporal values are currently normalized (`displayTime` is the fraction of the video) so both the spatial IoU and temporal distance are both unitless values.